spotlight/cross_validation.py

```diff
@@ -140,8 +140,8 @@ def user_based_train_test_split(interactions,
     if random_state is None:
         random_state = np.random.RandomState()
 
-    minint = np.iinfo(np.uint32).min
-    maxint = np.iinfo(np.uint32).max
+    minint = np.iinfo(np.int32).min
+    maxint = np.iinfo(np.int32).max
```

Comment:
I tried this too, but that didn't solve the issue for me, the alternative could be https://github.com/maciejkula/spotlight/pull/85/files#diff-491057b52bafec352697b7adc90d4786R146